app/pods/components/layout/md-footer/template.hbs

```diff
@@ -1,13 +1,22 @@
 <div class="footer-container">
-  <span><a class="btn btn-xs btn-warning"
+  <span class={{if overPercent "btn btn-xs btn-danger" "btn btn-xs btn-info"}}>
```

Comment:
Not sure we should render this as a button is it doesn't do anything.  Then again, it's so small probably doesn't matter. I would add a tooltip with a short explanation of what the percentage means.

Reply:
This is definitely a blunder on my behalf, I call it the copy and paste haste.  I have refactored it, for the sake of accessibility as a text element instead of a button.  Also added tooltip.
![image](https://user-images.githubusercontent.com/24277002/77062950-3d7ebd00-69b3-11ea-8f6a-b48f575cbc6e.png)